go-controller/pkg/config/config.go

```diff
@@ -57,16 +58,17 @@ var (
 
 	// Default holds parsed config file parameters and command-line overrides
 	Default = DefaultConfig{
-		MTU:               1400,
-		ConntrackZone:     64000,
-		EncapType:         "geneve",
-		EncapIP:           "",
-		EncapPort:         DefaultEncapPort,
-		InactivityProbe:   100000, // in Milliseconds
-		OpenFlowProbe:     180,    // in Seconds
-		MonitorAll:        true,
-		LFlowCacheEnable:  true,
-		RawClusterSubnets: "10.128.0.0/14/23",
+		MTU:                 1400,
+		ConntrackZone:       64000,
+		EncapType:           "geneve",
+		EncapIP:             "",
+		EncapPort:           DefaultEncapPort,
+		InactivityProbe:     100000, // in Milliseconds
+		OpenFlowProbe:       180,    // in Seconds
+		MonitorAll:          true,
+		LFlowCacheEnable:    true,
+		RawClusterSubnets:   "10.128.0.0/14/23",
+		OVSDBConnectTimeout: types.OVSDBTimeout,
```

Comment:
It looks like some spaces and orders were changed. I would avoid changing anything than what the patch intend to do. It helps the review.

Reply:
thanks @dougsland, only some spaces were added by gofmt, due to the new field `OVSDBConnectTimeout` is longer than the rest ones, gofmt added spaces for all the fields. 
I agree with you, but for this case, if I want to avoid the space change, I need either shorten the field name to make it less meaningful, or break gofmt convention, adding spaces seem more acceptable to me.

Reply:
to me it's fine. I just noticed a bunch of spaces added :) lgtm